bin/run_functions.js

```diff
@@ -177,6 +185,12 @@ function getQueryStringFromOptions(options) {
                 queryStringPart = 'tags=' + encodeURI(tags);
                 break;
             }
+            case 'callback':
+                queryStringPart = 'callback=' + encodeURI(options[key]);
+                break;
+            case 'endpoint':
```

Comment:
**endpoint** should be called hostAndPort, the way it is in our API

Reply:
All occurrences of endpoint have been replaced with hostAndPort.